src/App.tsx

```diff
@@ -1,8 +1,11 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import './App.scss';
+import { GoodList } from './component/GoodList';
+import cn from 'classnames';
+import { IListOfGoodsHooks } from './types/correctSort';
```

Comment:
The import `IListOfGoodsHooks` is not used in the component. If this type is intended to be used for props, ensure that it is correctly applied to the `App` component. Otherwise, consider removing the unused import.